www/js/store.js

```diff
@@ -10,7 +10,7 @@ function parseDataFile(filePath, defaults) {
   // which will be the case on the first application run.
   // `fs.readFileSync` will return a JSON string which we then parse into a Javascript object
   try {
-    return JSON.parse(fs.readFileSync(filePath));
+    return JSON.parse(fs.readFileSync(filePath, { flag: 'w' }));
```

Comment:
is `w` the correct flag? seems like `w+` is what we need according to the docs(https://nodejs.org/api/fs.html#fs_file_system_flags) but i could be wrong.